examples/basic/src/App.tsx

```diff
@@ -6,6 +6,7 @@ export const App = (): JSX.Element => {
       endpoint={import.meta.env.SEAM_ENDPOINT}
       publishableKey={import.meta.env.SEAM_PUBLISHABLE_KEY}
       userIdentifierKey={import.meta.env.SEAM_USER_IDENTIFIER_KEY}
+      disableCssInjection
```

Comment:
We have to put this in the example since when running locally the example should use the latest styles. We can keep the font injection.